src/components/UserProfile/UserProfileModal/UserProfileModal.jsx

```diff
@@ -142,6 +143,11 @@ const UserProfileModal = props => {
     }
   }
 
+  const adjustTextareaHeight = (textarea) => {
+    textarea.style.height = 'auto';
+    textarea.style.height = `${textarea.scrollHeight}px`;
```

Comment:
Hi Nathalia,
Everything looks good in the codebase but I would like to highlight that you are simply overwriting the same property on line 147 and 148. It will be better to get rid of the line 147 as it serves no purpose here. 
Please let me know if I have interpreted it wrongly.
Thanks!